kroxylicious-sample/sample-proxy-config.yml

```diff
@@ -16,8 +16,8 @@ virtualClusters:
       type: PortPerBrokerClusterNetworkAddressConfigProvider
       config:
         bootstrapAddress: localhost:9192
-    logNetwork: false
-    logFrames: false
+    logNetwork: true
```

Comment:
I think we should probably keep these set to false by default.

(though I presume they were committed unintentionally)

Reply:
It was unintentionally.. reverted